src/backend/lima.ts

```diff
@@ -2021,7 +2021,11 @@ class LimaKubernetesBackend extends events.EventEmitter implements K8s.Kubernete
   }
 
   async stop() {
-    await this.vm.execCommand({ root: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+    try {
+      await this.vm.execCommand({ root: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+    } catch (ex) {
+      console.debug('k3s stop failed: ', ex);
```

Comment:
Surely this warrants a `console.error`?

Reply:
I left it as a debug statement because the user probably won't even notice anything went wrong. But now that I know why this was failing CI, sure...

Reply:
ok